README.md

```diff
@@ -15,176 +19,103 @@
-
-4. `read()` : This function sent a GET request with specific Employeecode to read the information related to that particular Employeecode.
+## 🔄 Function Flow
 
-5. `delete()` : This function is used to delete the specific Employee Data.
+1. **`create_new_faceEntry()`**: Receives a POST request with an image and metadata. It extracts the face from the image, calculates the embeddings, and stores the data in the database.
 
-# Face Recognition Project
+2. **`Data()`**: Sends a GET request to the `/data` endpoint of the FastAPI app to retrieve the list of face entries from MongoDB.
 
-This project uses Flask, FastAPI,DeepFace and MongoDB to create a Face recognition system. It allows users to register face with associated metadata, update their information and also can delete their data.
+3. **`update()`**: Updates the details of a face entry in the database.
 
-### Get started
+4. **`read()`**: Sends a GET request with a specific `Employeecode` to retrieve related information.
 
-These instructions will get you a copy of the project up and running on your local machine for development.
+5. **`delete()`**: Deletes a specific employee's data from the database.
 
-### Prerequisites
-
-This project requires Python 3.7 or later.
-
-### Installing
-
-1. Clone this repository to your local system using link
-
-    ```bash
-    git clone "https://dev.azure.com/tmspl/FaceRec/_git/FaceRec-Employee-Enrollment.python"
-    ```
-
-2. Navigate to the project directory:
+## 🧪 Testing
 
-    ```bash
-    cd FaceRec
-    ```
-
-3. Install the required packages:
-
-    ```bash
-    pip install -r requirements.txt
-    ```
-
-
-### Running the Server
-To start FLask and FastAPI, run the given command:
+To run the tests for this project, use the following command:
 ```bash
-python main.py
+pytest
 ```
 
-## Project Structure
-
-- `requirements.txt`: Contains the Python dependencies for the project.
-- `API/`: Contains code of FastAPI application.
-- `FaceRec/`: Contain all files of HTML,CSS and flask application.
-- `main.py`: Contains code of to start FastAPI and flask together.
-
-## Database Schema
-
-1. Create new connection in MongoDB and Connect using given url
-   `URL: mongodb://localhost:27017/8000`
-
-2.  Create database using
-    Database name: `DatabaseName`
-    Collection name: `CollectionName`
-
-3.  Add data by importing json file:
-    From 'database.mongo' folder -> `{DatabaseName}.{CollectionName}.json`
-
-The database contains a `faceEntries` collection with the following schema:
-
-- `id`: A unique identifier for the face entry.
-- `Employeecode`: A unique  employee ID associated with the image.
-- `Name`: The name of the person in the image.
-- `gender`: The gender of the person.
-- `Department`: The department of the person
-- `time`: The time the face entry was created.
-- `embeddings`: The embeddings of the face image.
-- `Images`: Base64 encoded image file.
```

Comment:
This is also required but not in the readme.md but in the sphinx docs so can you add that as well